File: c#/program7.cs
using System;

class Digit
{
    public int No;

    public Digit(int no)
    {   
        this.No = no;
    }

    public int SumDigit()
    {
        int iDigit = 0,Sum = 0,temp = 0;

        temp = this.No;
        while(temp != 0)
        {
            iDigit = temp % 10;
            Sum = Sum + iDigit;
            temp = temp / 10;
        }
        return Sum;
    }
}

class program7
{
    public static void Main(string[] args)
    {
        int value = 0;

        Console.WriteLine("Enter number : ");
        value = int.Parse(Console.ReadLine());

        Digit obj = new Digit(value);
        int ret = obj.SumDigit();

        Console.WriteLine("Sum of the digits are : "+ret);
    }
}

File: java/program7.java

import java.util.Scanner;

class Digit
{
    public int iNo;

    public Digit(int X)
    {
        iNo = X;
    }

    public int SumDigit()
    {
        int iDigit = 0;
        int temp = 0, iSum = 0;

        temp = iNo;
        while(temp != 0)
        {
            iDigit = temp % 10;
            iSum = iSum + iDigit;
            temp = temp / 10;
        }
        return iSum;
    }
}

class program7
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        int iValue = sobj.nextInt();

        Digit obj = new Digit(iValue);
        System.out.println("Sum of the digits is : "+obj.SumDigit());
    }
}